src/CoreBundle/Resources/config/defaults.inc.php

```diff
@@ -1192,12 +1192,9 @@
 }
 
 /**
- * Path to the file that marks if maintenance mode is active. The current directory's hash is added to
- * distinguish multiple Chameleon instances on a shared host.
- * We use the SCRIPT_FILENAME variable because we do NOT want to resolve symlinks. Otherwise it would not be possible
- * to activate maintenance mode and then perform a deployment using atomic deploys, because the maintenance marker
- * would then change its name.
+ * Path to the file that marks if maintenance mode is active. The default value with "cmsdata" should take care
```

Comment:
I think the new comment doesn't address enough the requirements for the marker file, which are:
- Marker must survive deployment (if maintenance mode is active before deployment, it must still be active after deployment). I think this was the reason we used the temp dir before; we should check if this is still the case now and which requirements must be met by any deployment process.
- There must be one distinguishable marker per Chameleon installation on the same host (which is fulfilled if we store it in cmsdata, but it should be noted in the comment).
- In multi-node setups the same physical marker must be accessible to all nodes (which is only fulfilled for the default value if cmsdata is shared between nodes; the old description wasn't flawless, we need to make sure that our assumptions about file sharing and deployment are made explicit).

Reply:
ok, comment extended